net/client_grpc.go

```diff
@@ -157,3 +160,36 @@ func (p *proxyClient) Private(c context.Context, in *drand.PrivateRandRequest, o
 func (p *proxyClient) DistKey(c context.Context, in *drand.DistKeyRequest, opts ...grpc.CallOption) (*drand.DistKeyResponse, error) {
 	return p.s.DistKey(c, in)
 }
+
+//ControlClient is a struct that implement control.ControlClient and is used to request
+//a Share to a ControlListener on a specific port
+type ControlClient struct {
+	conn   *grpc.ClientConn
+	client control.ControlClient
+}
+
+// NewControlClient creates a client connection to the given target (localhost:8080)
+func NewControlClient() ControlClient {
+	var conn *grpc.ClientConn
+	conn, err := grpc.Dial("localhost:8080", grpc.WithInsecure())
```

Comment:
Put that address in a constant and refer to the constant instead of hardwriting it. Also move the port to something less used, more "random", as 8080 is often used for testing